tools/cloud-build/daily-tests/ansible_playbooks/test-mounts-and-partitions.yml

```diff
@@ -15,29 +15,42 @@
 ---
 
 - name: Get partition info
-  command: sinfo --format='%P' --noheader
+  ansible.builtin.command: sinfo --format='%P' --noheader
+  changed_when: False
```

Comment:
just out of curiosity, are you defining changed_when: false to make sure this is always executed?

Reply:
I am defining `changed_when` to False to indicate that, in fact, this command never changes anything. In the output of Ansible playbook it will always report OK. I do not believe that this impacts the default value of `failed_when`.